flink-contrib/flink-storm-compatibility/flink-storm-compatibility-core/src/main/java/org/apache/flink/stormcompatibility/api/FlinkTopologyContext.java

```diff
@@ -53,6 +58,10 @@ public FlinkTopologyContext(final StormTopology topology, final Map<Integer, Str
 				null, null);
 	}
 
+	public void setContext(final StreamingRuntimeContext context) {
```

Comment:
I would extend the constructor instead of `setContext`. Using the constructor is less error prone because it forced to provide the `RuntimeContext`. An additional call to `setContext` could be "forgotten" by a programmer.